src/client.cpp

```diff
@@ -46,6 +47,8 @@ std::string      g_szSMBusername        = DEFAULT_SMBUSERNAME;           ///< Wi
 std::string      g_szSMBpassword        = DEFAULT_SMBPASSWORD;           ///< Windows user password used to access share
                                                                          ///< Leave empty to use current user when running on Windows
 eStreamingMethod g_eStreamingMethod     = TSReader;
+TvDatabase::KeepMethodType	g_KeepMethodType	   = TvDatabase::Always;
+int							g_DefaultLifeTime	   = 100;							///< The default days which are configured in kodi
 bool             g_bFastChannelSwitch   = true;                          ///< Don't stop an existing timeshift on a channel switch
```

Comment:
Please check your whitespace settings. For Kodi, intentation tabs are 2 spaces, not tabs